tests/run.py

```diff
@@ -21,7 +21,6 @@
 
         'polymorphic',
 
-        'django.contrib.admin',
```

Comment:
This was causing:
`(admin.E402) 'django.contrib.auth.context_processors.auth' must be in TEMPLATES in order to use the admin application.`

... as the admin app isn't integrated yet, I've removed it. Hoping to add it back soon.